tiledb/bioimg/converters/axes.py

```diff
@@ -49,6 +49,32 @@ def transposed_array(self, a: np.ndarray) -> np.ndarray:
         return np.moveaxis(a, self.i, self.j)
 
 
+def transpose_array(a: np.ndarray, s: Sequence[T], t: Sequence[T]) -> np.ndarray:
```

Comment:
Does this need to be accessible outside the `Axes` dataclass ?

Reply:
Dropped `Axes.transpose` at https://github.com/TileDB-Inc/TileDB-BioImaging/pull/38/commits/5e5c5734fcbdd26ea905e3d6289e9b4a4deb06ea